cmd/configure.ac

```diff
@@ -319,6 +319,20 @@ AS_IF([test "x$with_unit_tests" = "xyes"], [
   AX_APPEND_COMPILE_FLAGS([-Werror], [CHECK_CFLAGS])
 ])
 
+AC_ARG_ENABLE([sanitize],
```

Comment:
Since this is only using asan at the moment, would it be better to just call it `asan` rather than `sanitize`?